targets/jm-touch-sensor-1.0/profile/touch.c

```diff
@@ -0,0 +1,9 @@
+#include "jdprofile.h"
+#include "jacdac/dist/c/capacitivebutton.h"
+#include "jacdac-c/inc/interfaces/jd_hw.h"
+
+FIRMWARE_IDENTIFIER(0x303b04aa, "JM TOUCH SENSOR V1.0");
```

Comment:
WHY SHOUTING?

Reply:
BECAUSE IT IS IMPORTANT PEOPLE KNOW IT IS A TOUCH SENSOR. 

Honestly, I think I just followed convention from another target :smile: